examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>We create a list of different types of gates you can apply: the X gate, which is a single qubit gate, as well as CNOT and CCNOT gates, which are multiple qubit gates.  </p><p> </p><p>We randomly choose which type of gates to apply and then apply these gates on randomly chosen qubits to see what kind of function has been created.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>